tests/test_bucket_operations.py

```diff
@@ -0,0 +1,23 @@
+import logging
+
+from framework import config
+from framework.connection import SSHConnection
+
+log = logging.getLogger(__name__)
+
+
+def test_bucket_operations(account_manager, bucket_manager, unique_resource_name, random_hex):
+    SSHConnection().connection
```

Comment:
if its not used, you can remove this and import as well

Reply:
Fixed